presto-product-tests/conf/docker/common/compose-commons.sh

```diff
@@ -23,7 +23,7 @@ function export_canonical_path() {
 
 source ${BASH_SOURCE%/*}/../../../bin/locations.sh
 
-export DOCKER_IMAGES_VERSION=${DOCKER_IMAGES_VERSION:-10}
+export DOCKER_IMAGES_VERSION=${DOCKER_IMAGES_VERSION:-904b5a3}
```

Comment:
See my note above: "The DOCKER_IMAGES_VERSION is temporary; I will update it to the appropriate release version once those docker images are pushed."